src/selic-entries.validations.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { selicEntries } from "./selic-entries";
+import fetchMock from "jest-fetch-mock";
+
+describe("selicEntries", () => {
```

Comment:
Where are the tests for these conditions stated in #7 ?

- initial can be empty
- final can be empty